src/app/pages/login/login-form/login-form.component.html

```diff
@@ -11,7 +11,7 @@
       <p class="notification">{{ notification }}</p>
       <div>
         <button class='launch' type='Submit'>Sign in</button>
-        <a href='#'>Forgot Password?</a>
+        <a routerLink='/reset-link'>Forgot Password?</a>
```

Comment:
Good work @joelethan, However, routerLink is an angular directive and It must be configured with a bindable input.
An example syntax
<a [routerLink]="['/routePath']" >wherever</a>

Reply:
@CryceTruly As far as I know, this can not be done on `<a></a>` tags. I have however implemented this feedback on buttons.